README.md

```diff
@@ -0,0 +1,146 @@
+<p align="center">
+    <img src="https://github.com/activeloopai/new-hub/blob/istranic/readme/docs/logos/hub_logo_compact.png" width="30%"/>
+    </br>
+    <h2 align="center">Dataset management for deep learning applications
```

Comment:
"Access your s3 datasets as if they were in memory from any machine"

Reply:
My goal for the tagline is to be all-encompassing and aspirational. 

"Access remote datasets as if they were in memory. From ANY machine" feels too specific, because Hub will be much more. 

However, I'm down for hunting for a better tagline. @McCrearyD can you tell me more about what you didn't like about the old one so I can understand the sentiment you want to put forward?